### PR TITLE
Chapter_13: User microposts

### PR DESCRIPTION
**Pull request by thanhchuc26**
Screenshot from 2021-05-27 01-05-41

### CODE REVIEW

**Comment by thanhchuc26** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -136,13 +140,68 @@ input {
...
+      padding: 5px 0;
+    }
+  }
+
+  .timestamp {
+    color: $gray;
+    display: block;
+    margin-left: 60px;
+  }
+}
```
Comment (vanvtt-0952) [inline app/assets/stylesheets/custom.scss]: thêm end-line nhé

File: app/controllers/microposts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+class MicropostsController < ApplicationController
+  before_action :logged_in?, only: [:create, :destroy]
+  before_action :correct_user, only: :destroy
+  def create
```
Comment (vanvtt-0952) [inline app/controllers/microposts_controller.rb:5]: line trống trước line này

File: app/helpers/microposts_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module MicropostsHelper
```
Comment (vanvtt-0952) [inline app/helpers/microposts_helper.rb]: Xóa file

File: app/views/layouts/_micropost_form.html.erb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<%= form_for @micropost do |f| %>
+  <%= render "layouts/error_messages", object: f.object %>
+  <div class="field">
+    <%= f.text_area :content, placeholder: t("micropost.new.micropost") %>
+  </div>
+  <%= f.submit t("micropost.create.title"), class: "btn btn-primary" %>
+  <span class="image">
+    <%= f.file_field :image, accept: ["image/jpeg", "image/gif", "image/png"]%>
```
Comment (vanvtt-0952) [inline app/views/layouts/_micropost_form.html.erb]: `["image/jpeg", "image/gif", "image/png"]` khai báo vào setting file

**Comment by thanhchuc26** [comment]
ready

**Comment by vanvtt-0952** [comment]
merged

**Pull request merged**